### DISCUSSION TITLE
How to shown LSPosed on different launcher

### DISCUSSION THREAD

**Discussion by A227193sa** [LSPosed/LSPosed#2472] [General]
I moved from Nova launcher to Home UI. Now LSPosed disappeared and the only way to open it is to go to Nova launcher and head to LSPosed icon. How can I show LSPosed on a different launcher? Thank you.

**Comment by rmiles7721** [comment]
I would like to know this as well. Or find out what activity or intent to run so I could create a shortcut from Tasker

**Comment by yujincheng08** [comment]
#2458
